app/controllers/api/searchables_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::SearchablesController < Api::BaseController
+  def users
+    results = PgSearch.multisearch(params[:prefix])
+    people_array = []
+    for result in results
+      if result.searchable_type == "Teacher"
+        people_array.push(Teacher.find(result.searchable_id))
+      else
+        people_array.push(Student.find(result.searchable_id))
+      end
+    end
+    render json: people_array
+  end
+
+  def roster
```

Comment:
Why is roster in searchables? shouldn't it be in admin controller? Looks like you're not doing anything related to searching in this function